src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -767,6 +754,19 @@ def __call__(
                 # compute the previous noisy sample x_t -> x_t-1
                 latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs, return_dict=False)[0]
 
+                if callback_on_step_end is not None:
+                    callback_kwargs = {}
+                    for k in callback_on_step_end_inputs:
+                        callback_kwargs[k] = locals()[k]
+                    callback_kwargs = callback_on_step_end(i, t, callback_kwargs)
```

Comment:
This could be something more descriptive like:
callback_**result** = callback_on_step_end(**step_idx**, t, callback_kwargs)

Because these aren't really keyword arguments, but results from the callback function.
And it makes sense to preserve the step_idx as it was before.

Reply:
Agree `callback_results` is a bit nicer here!